packages/langium/src/lsp/hover-provider.ts

```diff
@@ -14,6 +14,9 @@ import type { MaybePromise } from '../utils/promise-utils.js';
 import type { LangiumDocument } from '../workspace/documents.js';
 import type { DocumentationProvider } from '../documentation/documentation-provider.js';
 import { findDeclarationNodeAtOffset } from '../utils/cst-utils.js';
+import { isKeyword } from '../languages/generated/ast.js';
+import { createLangiumGrammarServices } from '../grammar/langium-grammar-module.js';
```

Comment:
Importing the `langium/grammar` code in general purpose code is not allowed (as it will always include the whole grammar language in all adopter bundles - which we definitely don't want). I'm actually a bit confused, we used to have a linting rule for that, which should fail the CI.

Reply:
Ok I understand. 
I just a pushed a new commit that fix this.